src/main.go

```diff
@@ -869,6 +878,10 @@ func (t *tester) executeStmt(query string) error {
 			return errors.Trace(err)
 		}
 
+		if !t.enableResultLog {
+			return nil
+		}
+
```

Comment:
How about move to after dumpToByteRows?
I'm not sure is whether the sql.Row a lazy object, if so, we may just ignore error that may happen during really use.

Reply:
Done